Stripe/STPBancontactSourceInfoDataSource.m

```diff
@@ -10,13 +10,14 @@
 
 #import "NSArray+Stripe_BoundSafe.h"
 #import "STPLocalizationUtils.h"
+#import "STPTextFieldTableViewCell.h"
 
 @implementation STPBancontactSourceInfoDataSource
 
 - (instancetype)initWithSourceParams:(STPSourceParams *)sourceParams {
     self = [super initWithSourceParams:sourceParams];
     if (self) {
-        self.title = STPLocalizedString(@"Bancontact Info", @"Title for form to collect Bancontact account info");
+        self.paymentMethodLabel = STPLocalizedString(@"Bancontact", @"Text for Bancontact payment method");
```

Comment:
ah, good point – I'll make this a `paymentMethodType` and get the label from there